gtfs-realtime/spec/en/reference.md

```diff
@@ -351,18 +351,26 @@ A geographic position of a vehicle.
 
 ## _message_ TripDescriptor
 
-A descriptor that identifies an instance of a GTFS trip, or all instances of a trip along a route. To specify a single trip instance, the trip_id (and if necessary, start_time) is set. If route_id is also set, then it should be same as one that the given trip corresponds to. To specify all the trips along a given route, only the route_id should be set. Note that if the trip_id is not known, then station sequence ids in TripUpdate are not sufficient, and stop_ids must be provided as well. In addition, absolute arrival/departure times must be provided.
+A descriptor that identifies a single instance of a GTFS trip. 
```

Comment:
We can perhaps add that it's an error if the TripDescriptor cannot be resolved into a single trip instance (0 trip or multiple instances)?

Reply:
Fixed in https://github.com/MobilityData/transit/pull/26/commits/28f357824e8530b775c613836f0f577e91d8ecb6